src/Ren/Compiler/Emit/ESModule.elm

```diff
@@ -166,6 +171,7 @@ declaration { public, name, type_, expr } =
 {-| -}
 type alias Builder t =
     { wrap : Pretty.Doc t -> Pretty.Doc t
+    , precedence : Maybe Int
     , expr : Pretty.Doc t
     }
```

Comment:
I'm not sure whether these properties should be ordered alphabetically.

Reply:
Not really that fussed about alphabetic ordering, just order them old-to-new IMO.